Client/game_sa/CCameraSA.cpp

```diff
@@ -210,7 +210,7 @@ CMatrix* CCameraSA::GetMatrix(CMatrix* matrix)
 
 void CCameraSA::SetMatrix(CMatrix* matrix)
 {
-    CMatrix_Padded* pCamMatrix = GetInterface()->Placeable.matrix;
+    CMatrix_Padded* pCamMatrix = GetInterface()->matrix;
     if (pCamMatrix)
     {
         pCamMatrix->vFront = matrix->vFront;
```

Comment:
This code had no effect previously. `matrix` was always `nullptr` because `Placeable` was misplaced